internal/snips/file.go

```diff
@@ -19,11 +21,13 @@ type File struct {
 	ID        string
 	CreatedAt time.Time
 	UpdatedAt time.Time
-	Size      uint64
 	Content   []byte
```

Comment:
```suggestion
	RawContent   []byte
```

We can make RawContent public so it's easy to use with the database pkg, and it's explicit that it's raw